google-cloud-core/src/main/java/com/google/cloud/ServiceOptions.java

```diff
@@ -77,17 +81,22 @@
   private static final String LIBRARY_VERSION = defaultLibraryVersion();
   private static final String APPLICATION_NAME =
       LIBRARY_VERSION == null ? LIBRARY_NAME : LIBRARY_NAME + "/" + LIBRARY_VERSION;
-  private static final long serialVersionUID = -5714029257168617973L;
 
   private static final String META_FILE_ROOT = "/META-INF/maven/";
   private static final String META_VERSION_KEY = "version";
+  private static final RetrySettings DEFAULT_RETRY_SETTINGS = getDefaultRetrySettingsBuilder()
+      .build();
+  private static final RetrySettings NO_RETRY_SETTINGS = getDefaultRetrySettingsBuilder()
```

Comment:
This would mean that those should be the "default" options for the generated clients and within the gax package in general, but it is not the case, since those are default only for gcj/manual clients.